source/game/field/obj/ObjectBase.hh

```diff
@@ -46,12 +48,14 @@ protected:
     void calcTransform();
 
     ObjectId m_id;
+    RailInterpolator *m_railInterpolator;
     BoxColUnit *m_boxColUnit;
     u16 m_flags;
     EGG::Vector3f m_pos;
     EGG::Vector3f m_rot;
     EGG::Vector3f m_scale;
     EGG::Matrix34f m_transform;
+    const System::MapdataGeoObj *m_globalObj;
```

Comment:
The `m_globalObj` name needs to be changed. It's referencing a very old name for `MapdataGeoObj`. I had originally suggested `m_mapObj`, but `m_geoObj` is fine too.

Reply:
I think I prefer `m_mapObj`